addons/mail/static/src/components/activity_mark_done_popover/activity_mark_done_popover.js

```diff
@@ -12,7 +12,6 @@ export class ActivityMarkDonePopover extends Component {
      * @override
      */
     setup() {
-        super.setup();
```

Comment:
Aren't these already possible to remove in Owl 1 if that is the guideline forward?
I would suggest to merge this change in a preliminary PR to ease the diff here.

Reply:
Note: it's probably not ok to remove in our components because of what is done in register messaging component.

Reply:
messaging component is not a problem: we extend the components when they are registered, not the other way around.